dprint.json

```diff
@@ -3,11 +3,16 @@
   },
   "markdown": {
   },
+  "exec": {
+    "associations": ["**/*.{rb,gemspec,rake}", "Rakefile", "Gemfile", "Steepfile"],
+    "rubocop": "bundle exec rubocop --autocorrect --stdin {{file_path}} --stderr",
```

Comment:
Using servermode makde faster, However closing vscode does not kill it. So it should be done in another PR or commit.